src/main/java/gift/dto/order/OrderRequest.java

```diff
@@ -5,13 +5,17 @@
 
 public class OrderRequest {
     public record Create(
+            @NotNull
+            Long productId,
             @NotNull
             Long optionId,
             @Min(1)
             @NotNull
             int quantity,
             @NotNull
```

Comment:
문자열의 경우에는 NotBlank를 보통 사용하는 것이 좋습니다!
```suggestion
            @NotBlank
```